lake/mock/mock.go

```diff
@@ -46,9 +48,11 @@ func NewPool(s string) (Pool, error) {
 		}
 	}
 	return Pool{
-		name:   name,
-		id:     fakeID(name),
-		layout: layout,
+		name:     name,
+		id:       fakeID(name),
+		branch:   branchName,
+		branchID: fakeID(branchName),
```

Comment:
Nit: Initialize fields in the order they're delcared.
```suggestion
		name:     name,
		branch:   branchName,
		id:       fakeID(name),
		branchID: fakeID(branchName),
```